assets/lang/ja.ini

```diff
@@ -547,6 +589,33 @@ tstr_AuxUIDragHintOvrlTheaterScreenBlocked=オーバーレイの位置は SteamV
+tstr_AuxUIQuickStartOverlayPropertiesHeader=オーバーレイプロパティ
+tstr_AuxUIQuickStartOverlayPropertiesBody=オーバーレイにはさまざまなカスタマイズオプションがあります。\n原点と表示モードは、オーバーレイが表示される場所とタイミングを設定します。\nオーバーレイが表示されるはずなのに見つからない場合は、これらのプロパティを確認してください。\n前者が役に立たない場合は、その位置をリセットすると役立つこともあります。\n\nオーバーレイをモーションコントローラーの近くにドラッグして、モーションコントローラーにドッキングすることもできます。
+tstr_AuxUIQuickStartOverlayPropertiesBody2=一部のプロパティはデフォルトで非表示になっています。\nすべてのオプションを表示するには、設定ウィンドウで「詳細設定を表示」を切り替えます。\n\nUIウィンドウの位置は、それぞれのボタンを長押しすることでリセットできます。\nダブルクリックや右クリックは、クイックショートカットトグルと同様にオーバーレイボタンでも機能します。
+tstr_AuxUIQuickStartSettingsHeader=設定
+tstr_AuxUIQuickStartSettingsBody=設定ウィンドウには、Desktop+ のすべてのグローバル設定が含まれています。\nオーバーレイバーの右端にある歯車のボタンから開けます。\n\nオーバーレイの場合と同様に、設定の変更は自動的に適用され、保存されます。
+tstr_AuxUIQuickStartProfilesHeader=プロフィール
+tstr_AuxUIQuickStartProfilesBody=Desktop+ には、2種類のプロファイルがあります。\n\nオーバーレイプロファイル:\n1つまたは複数のオーバーレイをそのプロパティとともに保存します。\nこれらは手動で読み込んだり、アクションやアプリケーション、プロファイルのイベントとして読み込むこともできます。\n\nアプリケーションのプロフィール:\nSteamVR アプリケーションの起動時に自動的に読み込まれるオーバーレイプロファイルやアクションを割り当てます。
+tstr_AuxUIQuickStartActionsHeader=アクション
+tstr_AuxUIQuickStartActionsBody=Desktop+ のアクションは、コントローラー入力やアプリケーションプロファイルに割り当てたり、オーバーレイのアクションバーに追加したりできる一連のコマンドです。\nアクションコマンドには、デスクトップ上の入力のシミュレートからオーバーレイレイアウトの変更までが含まれます。\n使い方次第では、外部アプリケーションを起動することもできます。\n\nDesktop+ には、確認できるサンプル アクションがいくつか付属しています。
+tstr_AuxUIQuickStartActionsBody2=アクションは、初期設定でアクティブ化されたオーバーレイ(オーバーレイに表示されるアクションボタン、オーバーレイ上のコントローラー入力)に該当しない場合は、フォーカスされたオーバーレイをターゲットにします。\n\nフォーカスされたオーバーレイは、最後にクリックされたオーバーレイです。\nデスクトップ上でシミュレートされた入力のように、ターゲットとなるオーバーレイは重要ではありません。それ以外の場合は、別のターゲットのオーバーレイ、あるいは複数のオーバーレイを指定するのが便利かもしれません。
+tstr_AuxUIQuickStartOverlayTagsHeader=オーバーレイタグは、そのタグのプロパティに基づいて自動的に割り当てられます。\n番号付けされたグローバルショートカットは、SteamVRの入力バインディングインターフェイスを介してコントローラ入力に割り当てる必要があります。
+tstr_AuxUIQuickStartSettingsEndBody=他にも多くの設定ができます。\nチェックボックスなど操作して試してみてください。\n\n行き詰まった場合は、ウィンドウの下部にある [初期設定に戻す] を押してください。\n特定の要素のみを復元するオプションもあります。
+tstr_AuxUIQuickStartFloatingUIHeader=フローティングUI
+tstr_AuxUIQuickStartFloatingUIBody=[フローティングUI]と呼ばれるインターフェースは、オーバーレイをポイントするたびに表示されます。\nダッシュボードでは、他のオーバーレイがポイントされていない場合に常に表示されます。\n\nフローティングUIには、ドラッグモードを切り替えてオーバーレイを移動できるようにするなど、オーバーレイを変更するための基本的なコントロールと、アクションボタンのカスタマイズ可能なセクションが含まれています。\n\nオーバーレイの種類によっては、他のコントロールも存在する場合があります。\nそれらの表示は、それぞれのオーバーレイプロパティでオーバーレイごとに切り替えることができます。
+tstr_AuxUIQuickStartDesktopModeHeader=デスクトップモード
+tstr_AuxUIQuickStartDesktopModeBody=Desktop+ は、デスクトップに表示されるウィンドウで設定することもできます。\nこれは、頻繁なキーボード入力が必要なタスクや、モーションコントローラーを接続せずにせってを変更したいときに便利です。\n\nほぼすべての機能は両方のモードで同じですが、Desktop+ VRキーボードのレイアウトをカスタマイズするために使用されるキーボードレイアウトエディターは、デスクトップモードでのみアクセスできます。\n\n[設定] ウィンドウの [トラブルシューティング] セクション、またはシステムトレイ/通知領域の Desktop+ アイコンから切り替えることができます。
+tstr_AuxUIQuickStartEndHeader=説明書 & ユーザーガイド
+tstr_AuxUIQuickStartEndBody=これだけで、Desktop+ を使い始めるのに十分なはずです。\n\n何か問題が起こったり、行き詰まったりした場合は、必ず説明書も確認してください。アクションバーのボタンをクリックすると、説明書を開けます。\n各オプションの詳細な説明と一般的な使用シナリオのステップバイステップガイドは、ユーザーガイドに記載されています。\n\nこのガイドを再度表示するには、[初期設定に戻す]ページの右下にある [クイックスタートガイドを表示] を押します。
```

Comment:
I've noticed a bit of inconsistency with quotation marks overall. I can't judge when it's appropriate to keep double quotes (") over「 and 」in Japanese text, so this might be no issue at all, but the square brackets in the Quick Start Guide are supposed to represent button labels, so I'd like to avoid changing the quotes into them.